### ISSUE TITLE
App stops to respond after opening once per ~10 attempts

### ISSUE THREAD

**Issue by antdanchenko** [status-im/status-react#3849]
### Description
App stops to respond after opening in random e2e tests, also the issue was reproduced manually during nightly build testing
Please see [logcat](https://assets.saucelabs.com/jobs/bb348d6093d3496b9a9040ebc688471a/logcat.log?download=true)
Similar issue has been seen before [3089](https://github.com/status-im/status-react/issues/3089)

#### Expected behavior
App doesn't stop to respond after opening

#### Actual behavior
App stops to respond after opening
![0001screenshot](https://user-images.githubusercontent.com/12611990/38675726-5c194fd4-3e61-11e8-8c2d-bda4513c30c3.png)

### Reproduction
- Install Status on the device or emulator (from scratch, without upgrading previous version)
- Open Status

### 
Additional Information
* Status version: `77e6f455`
* Operating System: Android 7.1

#### Logs

[logcat.txt](https://github.com/status-im/status-react/files/1902976/logcat.txt)

**Comment by status-open-bounty** [comment]
![Contract deploying](https://openbounty.status.im/img/deploying_contract.png)
Transaction: [0x22f6288ccf93a7ac32b29951f755b783077419b39de4231f0c8c41972c75128d](https://etherscan.io/tx/0x22f6288ccf93a7ac32b29951f755b783077419b39de4231f0c8c41972c75128d)

**Comment by Coffee-Boyy** [comment]
Based on these few lines in the log report, it looks like it may be an issue with either a [bug in the firebase job dispatcher](https://github.com/firebase/firebase-jobdispatcher-android/issues/101) or a version mismatch between the app's firebase sdk and the currently installed google play services version on the device. If the latter is the cause, this can be solved by updating `firebase-core to >= 15.0.0` and `google-services >= 3.3.0` based on [this article](https://developers.google.com/android/guides/versioning) which allows automatic sdk versioning based on the device's google play version.

```
04-11 23:47:24.553  4695  4695 W FA      : Service connection failed: ConnectionResult{statusCode=SERVICE_VERSION_UPDATE_REQUIRED, resolution=null, message=null}
04-11 23:47:24.565  1569  2213 D VoldConnector: SND -> {9 volume mkdirs /storage/emulated/0/Android/data/im.status.ethereum/cache/}
04-11 23:47:24.567  1569  1713 D VoldConnector: RCV <- {200 9 Command succeeded}
04-11 23:47:24.572  4695  4712 E FA      : Discarding data. Failed to send app launch
04-11 23:47:24.572  4695  4712 E FA      : Failed to get app instance id
04-11 23:47:24.572  4695  4712 E FA      : Failed to send current screen to service
```
